charts/velero/examples/vsphere/secret.yaml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: velero-aws-creds
+data:
+  # echo -n "[default]
+  # aws_access_key_id=ACCESSKEY
+  # aws_secret_access_key=SECRETACCESSKEY
+  # " | base64
+  cloud: W2RlZmF1bHRdCmF3c19hY2Nlc3Nfa2V5X2lkPUFDQ0VTU0tFWQphd3Nfc2VjcmV0X2FjY2Vzc19rZXk9U0VDUkVUQUNDRVNTS0VZCg==
```

Comment:
I would use stringData in this example here, then you do not need this comment to show whats encoded in this base64 string:
```yaml
stringData:
  cloud: |
    [default]
    aws_access_key_id=ACCESSKEY
    aws_secret_access_key=SECRETACCESSKEY
```

Reply:
Thanks, done